### ISSUE TITLE
Up-to-date library documentation is missing and wiki links are bad

### ISSUE THREAD

**Issue by jonstewart** [sleuthkit/sleuthkit#1823]
The wiki at sleuthkit.org designates this page as the entry point for library user's guide:

http://wiki.sleuthkit.org/index.php?title=Library_User%27s_Guide

It in turn links to the Doxygen-generated HTML. That documentation is out of date. First, the link on the wiki to Doxygen is for version 4.3 (http://www.sleuthkit.org/sleuthkit/docs/api-docs/4.3/). Second, the latest documentation is only for version 4.6 (http://www.sleuthkit.org/sleuthkit/docs/api-docs/4.6/). Links to documentation for 4.7 and 4.8 result in 404s. Please update release procedures to address.

**Comment by bcarrier** [comment]
We're in the process of killing the wiki. It was an experiment in crowd sourced editing that failed. It was FAR more popular with SPAMERs than forensics people (hundreds of accounts per day were being created).  So, we're slowly moving over any unique data from there into the various Doxygen-based documents that we have been maintaining. 

Based on your observations:
- That wiki page is gone.  It had no unique information that wasn't on other pages. References to it were updated to the doxygen page directly.
- I didn't realize the release script was not updating the version number in the docs, so the 4.6 doc was actually for 4.8. I just fixed the script and remade the 4.6, 4.6, and 4.8 docs.
- I also made it so that http://www.sleuthkit.org/sleuthkit/docs/api-docs/latest goes to the most recent build. 

If anyone would like to make changes to the library docs, they are located [here](https://github.com/sleuthkit/sleuthkit/tree/develop/tsk/docs). 

Thanks
